esmvaltool/config-references.yml

```
@@ -601,8 +605,9 @@ references:
   cox18nature: "Cox, P. M. et al., Nature, doi:10.1038/nature25450, 2018."
   davini12jclim: "Davini P., et al., J. Climate, doi: 10.1175/JCLI-D-12-00032.1, 2012"
   davini18: "Davini, P., Zenodo. doi:10.5281/zenodo.1237837, 2018"
-  deangelis15nat: "DeAngelis, A. M. et al., Nature, doi:10.1038/nature15770, 2015."
+  deangelis15nat: "DeAngelis et al., Nature, doi:10.1038/nature15770, 2015."
   demora2018gmd: "de Mora et al., Geosci. Model Dev., doi:10.5194/gmd-11-4215-2018, 2018."
+  docquier2017cryo: "Docquier et al., The Cryosphere, https://doi.org/10.5194/tc-11-2829-2017, 2017."
```

```suggestion
  docquier2017cryo: "Docquier et al., The Cryosphere, doi:10.5194/tc-11-2829-2017, 2017."
```
Sorry, just trying to keep the formatting consistent.